function throwError(y, z)

x

end